### PR TITLE
Update azure_rm_manageddisk related document

### PR DESCRIPTION
**Pull request by Fred-sun**
##### SUMMARY

Update azure_rm_manageddisk related document.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

azure_rm_manageddisk/azure_rm_manageddisk_facts
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by Fred-sun** [comment]
ready_for_review

File: lib/ansible/modules/cloud/azure/azure_rm_manageddisk.py [hunk 1]
```diff
@@ -75,41 +74,40 @@
...
             - read_write
-        version_added: 2.8
+        version_added: '2.8'
     tags:
         description:
             - Tags to assign to the managed disk.
+            - Format tags as 'key' or 'key:value'.
     zone:
         description:
-            - "Allowed values: 1, 2, 3, ''."
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_manageddisk.py:112]: In this line the values allowed were `1` or `2` or `3` or `.`. In the `choices` list below `.` is not listed, but `''` is. Can you confirm which list is correct?

File: lib/ansible/modules/cloud/azure/azure_rm_manageddisk_facts.py [hunk 2]
```diff
@@ -112,8 +116,7 @@
         tags:
             description:
                 - Tags to assign to the managed disk.
-            type: dict
-            sample: { "tag": "value" }
+                - Format tags as 'key' or 'key:value'.
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_manageddisk_facts.py]: I'd suggest keeping the old lines 115-116. This section describes the return data, so knowing that it's a dict and having the sample would be useful. The user cannot format the tags in the return data.

**Review by acozine — COMMENTED** [review]
Overall looks good. One question and one suggestion.

**Comment by acozine** [comment]
Thanks @Fred-sun.

**Pull request merged**